src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function ArticleCard({ article }) {
+  return (
+    <section>
+      <h2> Title: {article.title}</h2>
+      <h3>Author: {article.author}</h3>
+      <p>Topic: {article.topic}</p>
+    </section>
+  );
+}
```

Comment:
I like that you've separated out this card into it's own component
It would be good if you considered the styling a bit more, it's currently difficult to tell when one article card ends and another begins. Some sort of border or background colour may be helpful for this.
Also consider which information you're providing here - later we ask you to sort these articles by things such as date and comment count - so it would be good to have this info available from the get go 👀